### PR TITLE
Turns off draggability for Facility Locator map. closes #2479

### LINKED ISSUE #2479: UserVoice Idea: Using the facility locator on my phone in landscape mode
UX problem: Using the facility locator on my phone in landscape mode gets me trapped in panning around the map.  If I scroll down the page to look at the map while in landscape mode the map takes up the entire screen and when I try to scroll back up I end up just panning the map to the north. 

http://veterans.uservoice.com/forums/330333-vets-gov/suggestions/14745408-using-the-facility-locator-on-my-phone-in-landscap

### PR DESCRIPTION
**Pull request by webinista**

### CODE REVIEW

**Comment by webinista** [comment]
@ayaleloehr @crwallace Assigning to both of you, choose amongst yourselves as to who reviews it.

File: facility-locator/map.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ FacilityLocator = (function() {
       directionsStart,
       geolocated = false; // Used when determining starting location for driving directions
       zoomLevelToShowMarkers = 5;
-      infowindows = {};
+      infowindows = {},
```
Comment (ayaleloehr) [inline facility-locator/map.js]: The inconsistencies between commas and semicolons above really scares me, but that's not the fault of this PR.

File: facility-locator/map.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ FacilityLocator = (function() {
...
+      // Checks whether the matchMedia is a function (supported) and whether this is a small screen
+      isSmallScreen = !(typeof matchMedia === 'function') && (matchMedia('(max-width: 40.063em)').matches);
```
Comment (ayaleloehr) [inline facility-locator/map.js]: total curiosity and not blocking merging, but @webinista how did you come up with 40.063?

**Comment by ayaleloehr** [comment]
Thanks @webinista !  LGTM, but Travis issues mean we can't merge this yet. Robbie is working the travis problems, so will update this branch once that is in `master` and then merge.

File: facility-locator/map.js (hunk 2 above)
Comment (webinista) [inline facility-locator/map.js]: It's the current, calculated, small breakpoint used in our CSS. I copied-and-pasted from Chrome Dev Tools and didn't bother to round down to 40em.

File: facility-locator/map.js (hunk 1 above)
Comment (webinista) [inline facility-locator/map.js]: Pushed another commit that changes them to commas. They should be commas. Thanks for catching this.

**Comment by ayaleloehr** [comment]
@webinista this went green!  might not have to wait for Robbie's changes. Feel free to merge if you think it's good to go.

**Pull request merged**